### DISCUSSION TITLE
Lost the arctic vault badge after accidentally deleting email

### DISCUSSION THREAD

**Discussion by megamayoy** [community/community#21657] [New to GitHub]
<p>I had the arctic vault badge on my profile based on multiple contributions. here’re 2 of them</p><aside class="onebox githubpullrequest">
  <header class="source">
      <a href="https://github.com/AllAlgorithms/cpp/pull/80" target="_blank" rel="noopener nofollow ugc">github.com/AllAlgorithms/cpp</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Pull Request">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 12 16" aria-hidden="true"><path d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
  </div>

  <div class="github-info-container">
      <h4>
        <a href="https://github.com/AllAlgorithms/cpp/pull/80" target="_blank" rel="noopener nofollow ugc">Add DoublyinkedList implementation</a>
      </h4>

    <div class="branches">
      <code>AllAlgorithms:master</code> ← <code>megamayoy:add-doubly-linked-list</code>
    </div>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2018-10-04" data-time="13:06:25" data-timezone="UTC">01:06PM - 04 Oct 18 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/megamayoy" target="_blank" rel="noopener nofollow ugc">
          <img alt="megamayoy" src="https://user-images.githubusercontent.com/11338579/181066043-effcd152-4e55-4524-8be8-288e722462c9.jpeg" class="onebox-avatar-inline" width="20" height="20">
          megamayoy
        </a>
      </div>

      <div class="lines" title="1 commits changed 2 files with 340 additions and 0 deletions">
        <a href="https://github.com/AllAlgorithms/cpp/pull/80/files" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+340</span>
          <span class="removed">-0</span>
        </a>
      </div>
    </div>

  </div>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<aside class="onebox githubpullrequest">
  <header class="source">
      <a href="https://github.com/AllAlgorithms/cpp/pull/81" target="_blank" rel="noopener nofollow ugc">github.com/AllAlgorithms/cpp</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Pull Request">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 12 16" aria-hidden="true"><path d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
  </div>

  <div class="github-info-container">
      <h4>
        <a href="https://github.com/AllAlgorithms/cpp/pull/81" target="_blank" rel="noopener nofollow ugc">add circularlylinked list implementation</a>
      </h4>

    <div class="branches">
      <code>AllAlgorithms:master</code> ← <code>megamayoy:add-circularly-linked-list</code>
    </div>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2018-10-04" data-time="13:10:14" data-timezone="UTC">01:10PM - 04 Oct 18 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/megamayoy" target="_blank" rel="noopener nofollow ugc">
          <img alt="megamayoy" src="https://user-images.githubusercontent.com/11338579/181066044-722adc82-fd74-4f9d-846f-727d46171c35.jpeg" class="onebox-avatar-inline" width="20" height="20">
          megamayoy
        </a>
      </div>

      <div class="lines" title="1 commits changed 2 files with 348 additions and 0 deletions">
        <a href="https://github.com/AllAlgorithms/cpp/pull/81/files" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+348</span>
          <span class="removed">-0</span>
        </a>
      </div>
    </div>

  </div>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>After accidentally deleting the email that has my contributions, I added it back and checked my profile page again and the badge is gone. Is there a way to get it back?</p>

**Comment by jfullstackdev** [comment]
<aside class="quote" data-post="1" data-topic="175556">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/why-my-arctic-code-vault-contributor-badget-disappear/175556">Why my Arctic Code Vault Contributor badget disappear</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-account-questions/42"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #fb8532;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Having problems with your GitHub Account? Get Help here.">GitHub Account Questions</span></a>
  </div>
  <blockquote>
    I remember that I had that 

Arctic Code Vault Contributor 

name before, but after the achievement feature has come out, I found that I could not set the badge. :&lt; 
Any help pls? Or I’m out? :&lt;
  </blockquote>
</aside>

<p>maybe you have the same problem</p>
<p>he said:</p>
<p>“I just re-added the email that is linked to the Arctic Code Vault Program, and the badge re-appears.”</p>

**Comment by megamayoy** [comment]
<p>well, I think I have a different problem as I already added the email linked to my contributions after  deleting it and still the badge didn’t appear. Do you suggest that I delete the email and re-add it again?</p>

**Comment by jfullstackdev** [comment]
<p>yup, try that too,</p>
<p>what else can you remember before losing the badge? that might help in case you it will not appear after re-adding</p>

**Comment by megamayoy** [comment]
<p>Unfortunately, after deleting,adding and verifying the email. Nothing happened. I also set the email as my primary email still to no avail. :cry: I don’t remember anything that caught my attention. everything was fine until the email was accidentally deleted</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>The <a href="https://github.com/AllAlgorithms/cpp/pull/81/commits">commits</a> associated with those PRs aren’t associated with any GitHub accounts. (if it was, you’d be able to click on the username associated with the commit, and you can’t).</p>
<p>If you visit the commit url with <code>.patch</code> attached, you can see the email address associated with the commit.</p>
<p>for example:</p>
<p><code>https://github.com/AllAlgorithms/cpp/pull/81/commits/40769f1a1c5d17438975304b347990fe830ea2a3.patch</code></p>

**Comment by megamayoy** [comment]
<p>Thank you. I was adding the wrong email. after adding the correct email associated with my contributions, the badge appeared.</p>
